airbyte-integrations/bases/standard-destination-test/src/main/resources/format_reference.txt

```diff
@@ -0,0 +1,23 @@
+VALID FORMATS:
+
+"2021-1-1"
```

Comment:
could you add the ones from here too https://github.com/airbytehq/airbyte/issues/5439#issuecomment-906932101

more generally is this a comprehensive listing of ISO formats?

Reply:
It lacks only the dates with formats like this, should this be included as well?
 
"2021-W06-5" - Example with week date part: "Day 5 of Week 6"
"20210208" - Example of the short-date formats: "2021-02-08"
"2020-039 09" - Example with ordinal date part and hour time part: "Day 39 of 2020 at 9 a.m"

Those are not typical, but seems they are the part of the IOS8601 standard. 
Generally speaking, no matter the representation is, the string could be always format using the "pattern".

Reply:
> could you add the ones from here too [#5439 (comment)](https://github.com/airbytehq/airbyte/issues/5439#issuecomment-906932101)
> 
> more generally is this a comprehensive listing of ISO formats?

Made it like this: https://github.com/airbytehq/airbyte/issues/5439#issuecomment-912344778

Reply:
Added some more examples of the patterns.